app/javascript/utils/statusUtils.js

```diff
@@ -1,20 +1,30 @@
 import { find } from 'lodash'
 
-export const LEASE_UP_STATUS_OPTIONS = [
-  {value: 'No Status', label: 'No Status'},
+export const LEASE_UP_STATUS_DROPDOWN_OPTIONS = [
   {value: 'Processing', label: 'Processing', style: 'is-processing'},
-  {value: 'Disqualified', label: 'Disqualified', style: 'is-disqualified'},
-  {value: 'Approved', label: 'Approved', style: 'is-approved'},
-  {value: 'Lease Signed', label: 'Lease Signed', style: 'is-leased'},
-  {value: 'Waitlisted', label: 'Waitlisted', style: 'is-waitlisted'},
   {value: 'Withdrawn', label: 'Withdrawn', style: 'is-withdrawn'},
   {value: 'Appealed', label: 'Appealed', style: 'is-appealed'},
-  {value: null, label: 'All Status'}
+  {value: 'Waitlisted', label: 'Waitlisted', style: 'is-waitlisted'},
+  {value: 'Disqualified', label: 'Disqualified', style: 'is-disqualified'},
+  {value: 'Approved', label: 'Approved', style: 'is-approved'},
+  {value: 'Lease Signed', label: 'Lease Signed', style: 'is-leased'}
+]
+
+export const LEASE_UP_STATUS_FILTER_OPTIONS = [
+  'No Status',
```

Comment:
why the update to duplicate here? Seems like it'd be hard to track updates even if they're in the same file

Reply:
@akegan it was overwriting the status update dropdown to consolidate the two, but you're right it risks having to track in more than one place. I took a look again and just updated the code so it uses on constant and just appends and prepends the other two values when generating the filter -- see latest commit at f63e7a37d22bbd83d0777c3846c7b6338f1f7830.